fuzz/fuzz_targets/compile-maybe-invalid.rs

```diff
@@ -0,0 +1,10 @@
+#![no_main]
+
+use libfuzzer_sys::fuzz_target;
+use wasmtime::{Engine, Module};
+use wasmtime_fuzzing::wasm_smith::MaybeInvalidModule;
+
+fuzz_target!(|module: MaybeInvalidModule| {
+    let engine = Engine::default();
+    drop(Module::new(&engine, &module.to_bytes()));
+});
```

Comment:
This new fuzz target never initializes the logger or calls `log_wasm` on the generated bytes, so debugging failures for it will be annoying. Care to add a call to `log_wasm` here? (Which will internally initialize the `env_logger`)

Reply:
Oops excellent point